src/renderer/redirect.ts

```diff
@@ -1,23 +1,16 @@
 import { ActionResult } from '../models/response.ts';
 
-export function Redirect(url: string): ActionResult {
+export function Redirect(url: string, permanent: boolean = false): ActionResult {
```

Comment:
Do not touch this file, it is necessary to implement each redirect function https://httpstatuses.com/.